backend/app/utils/configs/utils.py

```
@@ -26,6 +26,8 @@
     validate_config_update,
 )
 
+from app.utils.exceptions.errors import not_found_error, conflict_error
```

Ruff (0.8.2)

`app.utils.exceptions.errors.conflict_error` imported but unused

Remove unused import: `app.utils.exceptions.errors.conflict_error`

(F401)